### ISSUE TITLE
train process issues

### ISSUE THREAD

**Issue by lingcon01** [Bhattacharya-Lab/EquiPNAS#3]
I repeated the training step and followed the scripts provided, but I found that the training loss rises suddenly. What could be the issue? Also, I noticed that the train_batch_size is set to 1 in your training process, and EGNN uses 12 layers. Could this easily lead to overfitting?
<img width="299" alt="image" src="https://github.com/user-attachments/assets/576c407f-37ef-4ae8-93b9-6aa92c95a16c" />
